python/python/glide/async_commands/server_modules/ft_options/ft_aggregate_options.py

```diff
@@ -0,0 +1,322 @@
+# Copyright Valkey GLIDE Project Contributors - SPDX Identifier: Apache-2.0
+from abc import ABC, abstractmethod
+from enum import Enum
+from typing import List, Mapping, Optional
+
+from glide.async_commands.server_modules.ft_options.ft_constants import (
+    FtAggregateKeywords,
+)
+from glide.constants import TEncodable
+
+
+class FtAggregateClause(ABC):
+    """
+    Abstract base class for the FT.AGGREGATE command clauses.
+    """
+
+    def toArgs(self) -> List[TEncodable]:
+        """
+        Get the arguments for the clause of the FT.AGGREGATE command.
+
+        Returns:
+            List[TEncodable]: A list of arguments for the clause of the FT.AGGREGATE command.
+        """
+        args: List[TEncodable] = []
+        return args
+
+
+class FtAggregateLimit(FtAggregateClause):
+    """
+    A clause for limiting the number of retained records.
+    """
+
+    def __init__(self, offset: int, count: int):
+        """
+        Initialize a new FtAggregateLimit instance.
+
+        Args:
+            offset (int): Starting point from which the records have to be retained.
+            count (int): The total number of records to be retained.
+        """
+        self.offset = offset
+        self.count = count
+
+    def toArgs(self) -> List[TEncodable]:
+        """
+        Get the arguments for the Limit clause.
+
+        Returns:
+            List[TEncodable]: A list of Limit clause arguments.
+        """
+        args = super().toArgs()
+        args.extend([FtAggregateKeywords.LIMIT, str(self.offset), str(self.count)])
+        return args
+
+
+class Filter(FtAggregateClause):
+    """
+    A clause for filtering the results using predicate expression relating to values in each result. It is applied post query and relate to the current state of the pipeline.
```

Comment:
Copy docs from java client - we should have same docs in all clients.
If you find them inaccurate - update.

Reply:
It is copied from JAVA client. Created a separate task to align docs for all commands. Will fix for both in that task.

Reply:
As discussed we will need to align everything for JAVA. Not updating here.